### PR TITLE
Fix LP debt 0 issue

### PR DESCRIPTION
**Pull request by KMKoushik**
# Task: LP debt is shown as zero,

## Description

Fixes # (issue)

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## Checklist

- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

### CODE REVIEW

**Review by alexisgauba — APPROVED** [review]

**Comment by yichiehliu** [comment]
image

I still have Error: execution reverted, not approved even though my LP management tx got approved, wondering why there is

**Comment by KMKoushik** [comment]
> image
> 
> I still have Error: execution reverted, not approved even though my LP management tx got approved, wondering why there is 🤔

@yichiehliu Which Tx got through? Did you give the correct LP id? Me and @alexisgauba got this issue because we are using someone else's address and we don't have approval. You should not get it if you use your own address!

**Comment by yichiehliu** [comment]
> > image
> > I still have Error: execution reverted, not approved even though my LP management tx got approved, wondering why there is 🤔
> 
> @yichiehliu Which Tx got through? Did you give the correct LP id? Me and @alexisgauba got this issue because we are using someone else's address and we don't have approval. You should not get it if you use your own address!

Umm I approved with id 224, I thought it was mine LP token id, and the error started to show up after I clicked approve button
https://ropsten.etherscan.io/tx/0x2deb79aadc20872fcdbabdd694febfddb2c1dfc7b61d1d5d8c556c5052233020

**Review by yichiehliu — APPROVED** [review]

File: packages/frontend/src/queries/uniswap/positionsQuery.ts
Comment (yichiehliu) [inline packages/frontend/src/queries/uniswap/positionsQuery.ts:83]: Have a q, do we need to put orderBy timeStamp in this query?

Comment (KMKoushik) [reply packages/frontend/src/queries/uniswap/positionsQuery.ts:83]: Nope not needed for now. If needed can add later

**Pull request merged**
